types/es6-shim/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "es6-shim": "0.34"
```

Comment:
Maybe I do not understand correctly, but I think `es6-shim` actually declares or extends types (if necessary) in the global scope (you do not have to explicitly import them). For example, `Array.prototype.values()` as defined in the ES6 specification p. 424 is not provided in node v7.9.0. When I run the tests without `require("es6-shim")` at the top of es6-shim-tests.js in node v7.9.0 it outputs this error:
```
.../DefinitelyTyped/types/es6-shim/es6-shim-tests.js:132
iterableIteratorOfPoint = arrayOfPoint.values();
                                       ^

TypeError: arrayOfPoint.values is not a function
...
```

Compiling the tests with tsc without having `require("es6-shim")` at the top gives:
```
$ node --max-old-space-size=8192 ~/.npm-packages/bin/tsc es6-shim-tests.ts --typeRoots .. | grep es6-shim
es6-shim-tests.ts(1,15): error TS2451: Cannot redeclare block-scoped variable 'require'.
es6-shim-tests.ts(20,5): error TS2451: Cannot redeclare block-scoped variable 'key'.
es6-shim-tests.ts(185,1): error TS2322: Type 'Promise<void>' is not assignable to type 'Promise<Point>'.
es6-shim-tests.ts(191,66): error TS2345: Argument of type 'PromiseLike<Point>' is not assignable to parameter of type 'Point | Thenable<P
oint>'.
es6-shim-tests.ts(196,37): error TS2345: Argument of type 'IterableShim<Point>' is not assignable to parameter of type '(Point | Thenable
<Point>)[]'.
es6-shim-tests.ts(198,31): error TS2345: Argument of type 'IterableShim<Point>' is not assignable to parameter of type '(Point | Thenable
<Point>)[]'.
es6-shim-tests.ts(199,1): error TS2322: Type 'Promise<{}>' is not assignable to type 'Promise<void>'.
es6-shim-tests.ts(202,34): error TS2345: Argument of type 'PromiseLike<Point>' is not assignable to parameter of type 'Point | Thenable<P
oint>'.
es6-shim-tests.ts(206,40): error TS2345: Argument of type 'Error' is not assignable to parameter of type 'Point'.
```
And of cause when you `require("es6-shim")`, you need to have it installed.

But that does not mean the `package.json` must be committed.